lib/GeoExt/data/MapFishPrintv3Provider.js

```diff
@@ -321,6 +321,9 @@ GeoExt.data.MapFishPrintv3Provider = Ext.extend(GeoExt.data.PrintProviderBase, {
                     matrices: []
                 });
                 if (layer.requestEncoding === "REST") {
+                    enc.baseURL = enc.baseURL
+                        .replace(/%7B/g, '{')
+                        .replace(/%7D/g, '}');
```

Comment:
Shouldn't we add as in the `getAbsoluteUrl` the `if (Ext.isSafari)` ?
To don't have eventual side effect other browser...